### PR TITLE
Liu Jie personal donation

### PR DESCRIPTION
**Pull request by garyyu**

### CODE REVIEW

**Comment by antiochp** [comment]
🚀

File: financials/income_log.csv [hunk 1]
```diff
@@ -7,4 +7,5 @@ income_id,date,type,source,currency,amount,exchange_rate,USD_equivalent,to_addre
...
 8,20181220,donation,Aurel,BTC,6.1169,4134.44,25289.96,3ChVP627KU5w4zu2rieFPF3wGXWQgmhvrs,https://www.blockchain.com/btc/tx/85251acf39fb801d04bd285ce91c115b875edc29a36103975212043f5e5ff2c4,Security audit funding campaign, https://www.grin-forum.org/t/security-audit-funding-campaign/962
 14,20180314,donation,GGM,GRIN,630.965700337,2.70,1703.61,https://donations.grin-tech.org,a1ef9ba5-4fd8-4ffb-bcde-19c26e21b366,Dev miner fee on Grinmint grincouncil account (mostly or only GGM),
-15,20190313,donation,Daisuke Shosaki,BTC,0.03939,3906.72,153.89,3ChVP627KU5w4zu2rieFPF3wGXWQgmhvrs,https://live.blockcypher.com/btc/tx/fcc646fa3d97af9d80dbecf5812d023c2edfd31b99f9cf592076554b7ac01496/,Japanese Grin supporter,
\ No newline at end of file
+15,20190313,donation,Daisuke Shosaki,BTC,0.03939,3906.72,153.89,3ChVP627KU5w4zu2rieFPF3wGXWQgmhvrs,https://live.blockcypher.com/btc/tx/fcc646fa3d97af9d80dbecf5812d023c2edfd31b99f9cf592076554b7ac01496/,Japanese Grin supporter,
+16,20190330,donation,Liu Jie,GRIN,500,3.24,1620,https://donations.grin-tech.org,refer the prize of Grin Chinese Naming event as the donation to Grin Fund,https://www.grin-forum.org/t/give-grin-a-chinese-name-grin/4376/105,
```
Comment (lehnberg) [inline financials/income_log.csv]: Perhaps it makes sense for `https://www.grin-forum.org/t/give-grin-a-chinese-name-grin/4376/105` to be in the URL column?

**Comment by lehnberg** [comment]
@ignopeverell could you confirm receipt of the 500 ツ?

**Comment by ignopeverell** [comment]
Confirmed, excellent!

**Pull request merged**